common/pom.xml

```diff
@@ -381,6 +392,18 @@
       </testResource>
     </testResources>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
Why is this needed?

Reply:
With Java 17's strong encapulation: [JEPS:403](https://openjdk.org/jeps/403) older tool or library that needs to access non-public fields and methods of java.* APIs by reflection requires us to use the add-opens option.